did_peer/tests/demo.rs

```diff
@@ -0,0 +1,56 @@
+use std::error::Error;
+
+use did_doc::schema::{
+    did_doc::DidDocument,
+    service::ServiceBuilder,
+    types::{uri::Uri, url::Url},
+    verification_method::{VerificationMethod, VerificationMethodType},
+};
+use did_doc_sov::extra_fields::{didcommv1::ExtraFieldsDidCommV1, ExtraFieldsSov, KeyKind};
+use did_parser::{Did, DidUrl};
+use did_peer::peer_did::{
+    numalgos::{numalgo2::Numalgo2, numalgo3::Numalgo3},
+    PeerDid,
+};
+
+#[test]
+fn demo() -> Result<(), Box<dyn Error>> {
```

Comment:
I would prefer avoiding test duplication by removing this (and potentially expanding the test macros, if you think it's worth it).

Reply:
I've moved this to `examples` directory which is quite common practice, added readme etc. Given it's now incorporated into readme as part of crate onboarding, I think you will find its existence more justified.